src/helpers/MdObjectHelper.ts

```diff
@@ -56,6 +67,64 @@ function buildArithmeticMeasureTitleProps(
     };
 }
 
+export const mdObjectToGeoPushpinBucketProps = (
+    config: IGeoConfig,
+    mdObject: VisualizationObject.IVisualizationObject,
+    filtersFromProps: AFM.ExtendedFilter[],
+): Partial<IGeoPushpinChartProps> => {
+    const hasVisualizationObjectContent = mdObject && mdObject.content;
+    if (!hasVisualizationObjectContent) {
+        return {};
+    }
+
+    const { content } = mdObject;
+    const { buckets: contentBuckets } = content;
+
+    const colorBucket = findBucketByLocalIdentifier(contentBuckets, COLOR);
+    const locationBucket = findBucketByLocalIdentifier(contentBuckets, LOCATION);
+    const segmentByBucket = findBucketByLocalIdentifier(contentBuckets, SEGMENT_BY);
+    const sizeBucket = findBucketByLocalIdentifier(contentBuckets, SIZE);
+    const tooltipTextBucket = findBucketByLocalIdentifier(contentBuckets, TOOLTIP_TEXT);
+
+    const color: IGeoPushpinChartBucketProps["color"] = colorBucket && colorBucket.items[0];
+    const location: IGeoPushpinChartBucketProps["location"] =
+        locationBucket && (locationBucket.items[0] as VisualizationObject.IVisualizationAttribute);
+    const segmentBy: IGeoPushpinChartBucketProps["segmentBy"] =
+        segmentByBucket && (segmentByBucket.items[0] as VisualizationObject.IVisualizationAttribute);
+    const size: IGeoPushpinChartBucketProps["size"] = sizeBucket && sizeBucket.items[0];
+    const tooltipText: VisualizationInput.IAttribute =
+        tooltipTextBucket && (tooltipTextBucket.items[0] as VisualizationObject.IVisualizationAttribute);
```

Comment:
```
interface IGeoPushpinChartBucketProps {
    location: VisualizationInput.IAttribute;
    size?: VisualizationInput.AttributeOrMeasure;
    color?: VisualizationInput.AttributeOrMeasure;
    segmentBy?: VisualizationInput.IAttribute;
    filters?: VisualizationInput.IFilter[];
    sortBy?: VisualizationInput.ISort[];
}
```
then
`const color: IGeoPushpinChartBucketProps[“color”] = colorBucket && colorBucket.items[0];`

why we don't use 
`const color: VisualizationInput.AttributeOrMeasure = colorBucket && colorBucket.items[0];`